nixos/modules/services/development/guix.nix

```diff
@@ -0,0 +1,118 @@
+{config, pkgs, lib, ...}:
+
+with lib;
+
+let
+
+  cfg = config.services.guix;
+
+  guix-binary = pkgs.stdenv.mkDerivation rec
```

Comment:
I'd definitely split the package out into its own file.

Reply:
Done. Now a maintainer is required. Would you be willing to fill that role @JohnAZoidberg?

Reply:
Yeah sure! I want to check out Guix - the UX of their commands seems really nice.

What made you switch?

Reply:
Thanks! Added you as maintainer.

> What made you switch?

Mostly curiosity to learn Scheme and increased interest in GNU projects and the whole philosophy behind them.